lib/timers.js

```diff
@@ -481,6 +481,7 @@ Timeout.prototype.unref = function() {
     }
 
     var handle = reuse(this);
+    delete handle._list;
```

Comment:
I thought V8 don't like deleting props prom objects, but it might be ok if it is not perf-sensitive...